tests/models/test_onnx.py

```diff
@@ -127,7 +132,7 @@ def test_error_if_input_sample_is_not_tensor(tmpdir):
                                          f'`Tensor`'):
         model.to_onnx(file_path, input_sample)
 
-
+@pytest.mark.skipif(not HAS_ONNX_RUNTIME, reason="onnxruntime wrongly configured")
```

Comment:
what does onnx have to do with logging metrics?
can this be a separate PR?

Reply:
onnx doesn't install properly on both Sean and I computers for some reasons. And I was breaking Discovery Tests process. So I added this skip if import fails. I will open a PR for it apart this one :)